docs/testcoverage.rst

```diff
@@ -0,0 +1,86 @@
+Test Coverage
+=============
+
+You can integrate Coverage.py with behave-django to find out the test coverage of your code.
+
+Dependencies
+------------
+
+At First, you should install Coverage.py dependency
+
+.. code-block:: bash
+
+    $ pip install coverage
+
+Environment.py
+--------------
+
+In ``environment.py``, add the code snippet below in the ``before_all`` function to start measuring test coverage:
+
+.. code-block:: python
+
+    import coverage
+
+    def before_all(context):
+        cov = coverage.Coverage()
+        cov.start()
+        context.cov = cov
+
+Then write below code to end up measuring test coverage.
+You can save the coverage result on html format.
+
+.. code-block:: python
+
+    def after_all(context):
+        cov = context.cov
+        cov.stop()
+        cov.save()
+        cov.html_report(directory="./cov")
```

Comment:
Same here: Couldn't we simply integrate that code [in our management command](https://github.com/behave/behave-django/blob/main/behave_django/management/commands/behave.py#L156-L157) and allow our users to activate the feature via a command line switch?

Then, the coverage report should probably not be written to an explicit location. The developer using the feature might want to control this via [coverage settings](https://coverage.readthedocs.io/en/latest/config.html) from the outside.